exercises/practice/palindrome-products/.meta/example_2.rb

```diff
@@ -0,0 +1,52 @@
+class Palindrome
+  attr_reader :value, :factors
+  private
+  def initialize(palindrome)
+    @value, @factors = palindrome
+  end
+end
```

Comment:
This class is not styled like the other class.  We should have vertical spacing and organization that is consistent, because inconsistencies draw attention (which is sometimes wanted) and suggests that there is "something different here, pay attention" when there really is not.